packages/web/app/src/lib/kit/types/json.ts

```diff
@@ -0,0 +1,19 @@
+export namespace JSON {
+  export const encode = <value extends Value>(value: value): string => {
+    return globalThis.JSON.stringify(value);
+  };
+
+  export const encodePretty = <value extends Value>(value: value): string => {
+    return globalThis.JSON.stringify(value, null, 2);
```

Comment:
Same, what's the difference between:

```js
Hive.kit.JSON.encodePretty(value):
// vs
JSON.stringify(value, null, 2)
```

Imo it's unnecessary layer on top of another layer just to perform JSON serialization

Reply:
https://github.com/graphql-hive/console/pull/6378#discussion_r1920311753

Reply:
We don't need two threads for this conversation. I am resolving this one.